pyaedt/emit.py

```diff
@@ -1,16 +1,242 @@
-from __future__ import absolute_import  # noreorder
+from __future__ import absolute_import
 
+import os
+import sys
+from importlib import import_module
+
+from pyaedt import generate_unique_project_name
 from pyaedt.application.AnalysisEmit import FieldAnalysisEmit
+from pyaedt.generic.general_methods import pyaedt_function_handler
+
+# global variable used to store module import
+mod = None
+
+
+class Result:
+    """
+    Provides the Result object.
+
+    Examples
+    --------
+    Create an instance of Result.
+    >>> aedtapp.results = Result()
+    >>> mode = Emit.tx_rx_mode().rx
+    >>> radio_RX = aedtapp.results.get_radio_names(mode)
+    """
+
+    @pyaedt_function_handler()
+    def __init__(self, emit_obj):
+        self.__result_loaded = False
+        self.emit_api = mod.EmitApi()
+        self.revisions_list = []
+        self.location = emit_obj.oproject.GetPath()
+        self.curr_design = 0
+
+    @pyaedt_function_handler()
+    def set_result_loaded(self):
+        """
+        set __result_loaded status to True.
+
+        Returns
+        -------
+
+        Examples
+        ----------
+        >>> aedtapp.results.set_result_loaded():
+        """
+        self.__result_loaded = True
+
+    @pyaedt_function_handler()
+    def get_result_loaded(self):
+        """
+        Return __result_loaded status.
+
+        Returns
+        -------
+        :class:`bool`
```

Comment:
```suggestion
        bool
            ``True`` when successful, ``False`` when failed.
```
This is our standard statement for when a Boolean value is returned.